_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -157,6 +161,67 @@ The [computed][] value of the `letter-spacing` property of this `p` element is *
 </p>
 ```
 
+#### Passed Example 9
+
+This `p` element only has one text node descendant, and it is [wide enough][] because its parent is the `span` element, with a `letter-spacing` of `2px`, more than `0.12` times the `font-size` of `10px`.
+
+```html
+<p style="font-size: 24px; letter-spacing: 2px !important">
+	<span style="font-size: 10px;">
+		The toy brought back fond memories of being lost in the rain forest.
+	</span>
+</p>
+```
+
+#### Passed Example 10
+
+This `p` element has no text node descendant and therefore matches the [spaced text](#24afc2:spaced-text) condition.
+
+```html
+<p style="letter-spacing: 1px !important; border-top: 1px solid black;">
+	<!-- empty div, border to make it "visible" -->
+</p>
+```
+
+#### Passed Example 11
+
+This `p` element only has one text node descendant, and it is [empty](#24afc2:empty).
+
+```html
+<p style="letter-spacing: 1px !important; border-top: 1px solid black;">
+	&nbsp:<!-- empty div, border to make it "visible" -->
+</p>
+```
+
+#### Passed Example 12
+
+Both this `p` and `span` elements have a single text node descendant, whose parent (the `span` element) is [wide enough][].
+
+```html
+<p style="font-size: 24px; letter-spacing: 2px !important">
+	<span style="letter-spacing: 3px !important;">
+		The toy brought back fond memories of being lost in the rain forest.
+	</span>
+</p>
+```
+
+#### Passed Example 13
+
+This `p` element has three text node descendants. The first one is [empty](#24afc2:empty); the second one is [wide enough][] because of the smaller `font-size` on its parent; and the last one is [wide enough][] because of the `letter-spacing` specified on its parent. The third `span` element has a [wide enough][] text node descendant.
```

Comment:
I feel like it would be good to specifically highlight there are two test targets in this example. Also, added the reference to the cascade condition.

```suggestion
This `p` element has three text node descendants. The first one is [empty](#24afc2:empty); the second one is [wide enough][] because of the smaller `font-size` on its parent; and the last one is [wide enough][] because of the `letter-spacing` specified on its parent, meeting the [cascade](#24afc2:cascade) condition. The third `span` element, which is also applicable, has a [wide enough][] text node descendant.
```

Reply:
For the `p` target, the text inside the third `span` is actually meeting both the "cascade" condition **and** the "wide enough" one (because the parent of this text node, the `span`, has a used `letter-spacing` of 3.6px, and a computed `font-size` of 30px; and 30×1.2 ⩽ 3.6).
I felt that "wide enough" is easier to understand 😅 But I guess we can keep both…